pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/subjects/SierraMeetingSubjects.scala

```diff
@@ -23,29 +24,27 @@ import weco.sierra.models.marc.VarField
 //
 object SierraMeetingSubjects
     extends SierraSubjectsTransformer
-    with SierraAgents {
+    with SierraAgents
+    with ConceptsTransformer {
 
   val subjectVarFields = List("611")
 
   val labelSubfields = List("a", "b", "c")
 
   def getSubjectsFromVarFields(bibId: SierraBibNumber,
-                               varFields: List[VarField]) =
+                               varFields: List[VarField]): Output =
     varFields.flatMap { varField =>
       createLabel(varField, subfieldTags = List("a", "c", "d")) match {
         case "" => None
```

Comment:
This is the only one of these SubjectTransformers that does this.
Others look like they would create a subject with an empty label.

I think there is scope for refactoring this for harmony.  Probably so that getSubjectsFromVarFields is in the base trait, and it is only the creation of an individual subject that changes by object.